src/content/blog/2024-11-05-ilp-packet-lifecycle.mdx

```diff
@@ -0,0 +1,59 @@
+
+A prospective debitor will use this standard to create what Open Payments refers to as an _outgoing payment_ on the Rafiki instance hosting the wallet address they want to charge, and an _incoming payment_ on Rafiki instance hosting the wallet address they want to send the payemnt to.
+The incoming and outgoing payment will be created by a series of requests against those Rafiki instances that set the terms of the payment, such as how much to charge the sender and in what currency, and in the case of the outgoing payment, acquire consent from the owner of that wallet address to create that charge.
+Once that is complete, Rafiki can begin to fulfill the terms of this payment using Interledger.
+
+On a more technical point, ILP packets may be involved before an outgoing payment is even created. As one of the requests prior to creating an outgoing payment, the debitor may also request a _quote_ from that Rafiki instance to account for any fees or conversion rates that may be incurred. To acquire this quote, the instance will send a ILP packet accross the network to the destination. This packet will need to traverse each hop on the network as each node's fees and rates won't be visible to the sender without doing so.
+
+### Preparing the Packet
+Now that the sending Rafiki instance has an outgoing payment created against one of its wallet addresses, it can pass the terms of that payment into an Interledger payment. First, it uses the Simple Payment Setup Protocol(link) to acquire an ILP address for the receiver, and a shared secret that is used to generate a _condition_. A condition is a hash that can only be generated with a secret that is available only to the receiver and the sender.
+Using the receiving ILP address, the condition, the receiver, and send amount described in the outgoing payment, Rafiki creates a connection over ILP to the receiver using STREAM (link).
+STREAM is a transport-layer protocol for Interledger that is responsible for breaking a payment down into the aforementioned packets and sends them _en masse_ until the payment is fulfilled.
+
+### Responding with Fulfill or Reject
+In order to complete a payment, the receiver will need to handle each packet with either a fulfillment or a rejection, so that the sender knows whether or not a given packet was successfully sent or not.
+The fulfill/reject responses are analagous to how a HTTP request may receive a response or be rejected.
+Barring bad configurations or other unusual cases, the receiver will generally fulfill a packet _as long as the condition it was sent with can be generated with the receiver's copy of the shared secret, and as long as the amount received already from prior fulfilled packets is below the total expected by the receiver._
```

Comment:
Maybe add a mention of how this makes packets secure so they can't be intercepted.